tests/json-patch-tests/tests.json

```diff
@@ -136,7 +136,7 @@
       "doc": ["foo", "sil"],
       "patch": [{"op":"add", "path": "/3", "value": "bar"}],
       "error": "index is greater than number of items in array" },
-      
+     
```

Comment:
Don't change anything here. This is a git subtree that contains the contents of https://github.com/json-patch/json-patch-tests

But you can make an upstream pull request and I will update the subtree, if it get's accepted 😃

Reply:
Seems upstream dead. But do you want to make it a submodule? It better tracks files which is imported from other repos.